introduction/introduction.md

```diff
@@ -0,0 +1,15 @@
+This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# Introduction
+
+This Learning Path gives an introduction to inner source.
+Inner source is the application of open source practices and principles to software development within the enterprise.
+Inner source software remains proprietary to the company, but within it is open for anyone to use it and contribute to it.
+This strategy enables wide and effective collaboration, producing software that is responsive and nimble to the changing needs of its many internal stakeholders.
```

Comment:
Hmm ... I think InnerSource/inner source is not primarily about the properties of the software that it produces (nimbleness or responsiveness) but rather about the process of creating it. I propose to remove that and add more info on the quality of the process and the overall outcome for the organisation (more efficiency, better collaboration) and the involved individuals (learning) instead.

Reply:
We've iterated on this paragraph a number of times in this PR, but I feel like my effectiveness in incorporating your great thoughts is waning.  If this paragraph needs further attention, can you open a PR with suggested text that you'd like to see?  Thanks